Tools/wasm/emscripten/__main__.py

```diff
@@ -215,12 +215,29 @@ def configure_emscripten_python(context, working_dir):
     exec_script = working_dir / "python.sh"
     exec_script.write_text(
         dedent(
-            f"""\
+            """\
             #!/bin/sh
 
+            # Macs come with a defective fork of coreutils so feature detect and
+            # work around it.
+            if which grealpath > /dev/null; then
+                # It has brew installed gnu core utils, use that
+                REALPATH="grealpath -s"
+            elif which readlink > /dev/null && realpath --version | grep GNU > /dev/null; then
```

Comment:
`--version` isn't a valid option for FreeBSD's realpath, so this generates output on stderr.
```suggestion
            elif which realpath > /dev/null && realpath --version 2&>1 | grep GNU > /dev/null; then
```